SU-HSDO/suhumsci
# SHS-5061: Upgrade text format on postcard text area

# READY FOR REVIEW

## Summary
Replaces Postcard component Body field format `Minimal HTML` with `Minimal HTML with Styles`  

## Steps to Test
1. Check that `Minimal HTML with Styles` format exists and has correct permissions `/admin/config/content/formats`
2. Ensure configuration for `Minimal HTML with Styles` matches `Minimal HTML`, but with the `Styles` button added to the toolbar `/admin/config/content/formats/manage/minimal_html_with_styles`
3. Check that ` Minimal HTML with Styles` option is the only one checked in `/admin/structure/paragraphs_type/hs_postcard/fields/paragraph.hs_postcard.field_hs_postcard_body`
4. Create content (e.g., flexible page) and embedded the Postcard component
5. The wysiwyg editor in the `Body` field should have the ability to use `Styles`

## Notes
- _Do not merge until design work has been completed_

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
